package.json

```diff
@@ -3,7 +3,10 @@
   "id": "firefox-voice@mozilla.org",
   "description": "This is a WebExtension that allows the browser to be voice-controlled.",
   "private": true,
-  "version": "0.19.0",
```

Comment:
why removed this line?

Reply:
This is also what's causing the build to fail: https://circleci.com/gh/mozilla/firefox-voice/2588